app/Post_Types/Diviner_Field/PostMeta.php

```diff
@@ -98,8 +109,61 @@ public function add_post_meta() {
 			->where( 'post_type', '=', Diviner_Field::NAME )
 			->add_fields( array(
 				$this->get_field_taxonomy_type(),
+				$this->get_field_taxonomy_singular_label(),
+				$this->get_field_taxonomy_plural_label(),
+				$this->get_field_taxonomy_slug(),
+			))
+			->set_priority( 'low' );
+
+		$this->container = Container::make( 'post_meta', 'Custom Post Type Field Variables' )
+			->where( 'post_type', '=', Diviner_Field::NAME )
+			->add_fields( array(
+				$this->get_field_CPT_ID(),
+				$this->get_field_CPT_Label(),
+				$this->get_field_CPT_SLUG(),
 			))
 			->set_priority( 'low' );
+
+		$this->container = Container::make( 'post_meta', 'Select Field Variables' )
+			->where( 'post_type', '=', Diviner_Field::NAME )
+			->add_fields( array(
+				$this->get_field_select_options(),
+			))
+			->set_priority( 'low' );
+	}
+
+	public function get_field_select_options() {
+		return Field::make( 'complex', self::FIELD_SELECT_OPTIONS )
+			->add_fields( array(
+				Field::make( 'text', self::FIELD_SELECT_OPTIONS_LABEL, 'Drop down label' ),
+			) );
+	}
+
+	public function get_field_CPT_ID() {
```

Comment:
why this mix, here and below, of upper and lower case?

Reply:
fixed